src/GovUk.Education.ExploreEducationStatistics.Admin/Services/UserManagementService.cs

```diff
@@ -305,5 +405,77 @@ await _usersAndRolesDbContext.UserRoles
                         .OnSuccess(() => _userRoleService.AddGlobalRole(userId, roleId));
                 });
         }
+
+        private async Task<Either<ActionResult, Unit>> ValidateEmail(string email)
+        {
+            if (string.IsNullOrWhiteSpace(email))
```

Comment:
As with `ValidateUserDoesNotExist`, this is preexisting code that I've refactored. So I've left it the way it was originally.

Since the plan now is to change email validation in a different ticket, I'm inclined to leave this for now.

Reply:
Disagree - it's super easy to fix this by adding the `[EmailAddress]` annotation to the request. In fact, this validation is useless because it doesn't even actually validate the email properly!

Reply:
I was considering the time consuming thing to be understanding what the EmailAddress annotation does, whether it is adequate, and removing all the existing email validation across all our services.

I'll add it to the places you've suggested.

Reply:
Incidentally, looking into it, this is the checks that the annotation does:
```
            if (value == null)
            {
                return true;
            }

            if (!(value is string valueAsString))
            {
                return false;
            }

            // only return true if there is only 1 '@' character
            // and it is neither the first nor the last character
            int index = valueAsString.IndexOf('@');
            
            return
                index > 0 &&
                index != valueAsString.Length - 1 &&
                index == valueAsString.LastIndexOf('@');
```

So I think we may want something more thorough and this. Searching online, it seems others tend towards some crazy regex.